test/helpers

```diff
@@ -2,9 +2,50 @@
 
 ### Helpers functions
 
-test::helpers::deploy() {
-    # Deploys a PHP app step by step and tests that everything is working as
-    # expected.
+test::helpers::classic_deploy() {
+    # Deploys a classic PHP app step by step and tests that everything is
+    # working as expected.
+    # To do so, this function runs the buildpack scripts and mimicks the
+    # platform behavior wherever needed.
+    #
+    # Globals:
+    #   STD_OUT
+    #   default_modules
+    #
+    # Arguments:
+    #   $1 > $php_version
+    #      See `test::helpers::php`.
+    #
+    #   $2 > $nginx_version
+    #      See `test::helpers::nginx`
+
+    local php_version="${1}"
+    local nginx_version="${2}"
+
+    # Test bin/detect:
+    test::helpers::detect "PHP (classic)"
+
+    # Test bin/compile:
+    test::helpers::compile
+
+    # Switch environment:
+    test::helpers::enter_prod
+
+    # Test PHP version:
+    test::helpers::php "${php_version}"
+
+    # Test PHP default modules:
+    test::helpers::default_php_modules
+
+    # Test Nginx version:
+    test::helpers::nginx "${nginx_version}"
+
+    # No Composer
+}
+
+test::helpers::composer_deploy() {
```

Comment:
suggestion(non-blocking): with both `composer_deploy` and `classic_deploy` sharing most of the code base, we could create a common function named `test::helpers::deploy_common` that is called by these functions.

Reply:
Right, applied in https://github.com/Scalingo/php-buildpack/pull/409/commits/4217951328756a3b8507d264f1207058369429db